gcloud-java-contrib/gcloud-java-nio/pom.xml

```diff
@@ -50,11 +50,6 @@
       <scope>provided</scope> <!-- to leave out of the all-deps jar -->
```

Comment:
I've removed this line, and `mvn clean compile test` seems to still work for me.